packages/terra-badge/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+* Added
+  * Added hidden text to announce the intent of the badge
+* Changed
+  * Removed `aria-hidden` from the textContent to announce text of badge
```

Comment:
for consistency can we add line break after each log
```suggestion
* Added
  * Added hidden text to announce the intent of the badge

* Changed
  * Removed `aria-hidden` from the textContent to announce text of badge
```

Reply:
Done https://github.com/cerner/terra-core/pull/3909/commits/fdb89cecb479d1e29495d6b4a10f08faa9e53b86